shader/path_tiling.wgsl

```diff
@@ -62,19 +62,19 @@ fn main(
         let idxdy = 1.0 / (dx + dy);
         var a = dx * idxdy;
         let is_positive_slope = s1.x >= s0.x;
-        let sign = select(-1.0, 1.0, is_positive_slope);
-        let xt0 = floor(s0.x * sign);
-        let c = s0.x * sign - xt0;
+        let sign_ = select(-1.0, 1.0, is_positive_slope);
```

Comment:
Makes sense but I'm not in love. I think I might prefer `xsign` (or possibly `x_sign` as it represents the sign of dx.